apps/plotting.py

```diff
@@ -750,178 +780,184 @@ def draw_lightcurve(switch: int, pathname: str, object_data, object_upper, objec
+                hovertemplate_upper = r"""
+                <b>Upper limit</b>: %{y:.2f}<br>
+                <b>%{xaxis.title.text}</b>: %{x|%Y/%m/%d %H:%M:%S.%L}<br>
+                <b>mjd</b>: %{customdata}
+                <extra></extra>
+                """
+                idx = pdf_upper['i:fid'] == fid
+                figure['data'].append(
+                    {
+                        'x': dates_upper[idx],
+                        'y': pdf_upper['i:diffmaglim'][idx],
+                        'mode': 'markers',
+                        'name': '',
+                        'customdata': pdf_upper['i:jd'].apply(lambda x: x - 2400000.5)[idx],
+                        'hovertemplate': hovertemplate_upper,
+                        "legendgroup": "{} band upper".format(fname),
+                        'legendrank': 101 + 10*fid,
+                        'marker': {
+                            'color': color,
+                            'symbol': 'triangle-down-open'
+                        },
+                        # 'showlegend': False
+                    }
+                )
+
+            # Lower-quality data points
+            if not pdf_upperv.empty:
+                # <b>candid</b>: %{customdata[0]}<br> not available in index tables...
+                hovertemplate_upperv = r"""
+                <b>%{yaxis.title.text} (low quality)</b>: %{y:.2f} &plusmn; %{error_y.array:.2f}<br>
+                <b>%{xaxis.title.text}</b>: %{x|%Y/%m/%d %H:%M:%S.%L}<br>
+                <b>mjd</b>: %{customdata}
+                <extra></extra>
+                """
+                idx = pdf_upperv['i:fid'] == fid
+                figure['data'].append(
+                    {
+                        'x': dates_upperv[idx],
+                        'y': pdf_upperv['i:magpsf'][idx],
+                        'error_y': {
+                            'type': 'data',
+                            'array': pdf_upperv['i:sigmapsf'][idx],
+                            'visible': True,
+                            'width': 0,
+                            'opacity': 0.5,
+                            'color': color
+                        },
+                        'mode': 'markers',
+                        'customdata': pdf_upperv['i:jd'].apply(lambda x: x - 2400000.5)[idx],
+                        'hovertemplate': hovertemplate_upperv,
+                        "legendgroup": "{} band".format(fname),
+                        'marker': {
+                            'color': color,
+                            'symbol': 'triangle-up'
+                        },
+                        'showlegend': False
+                    }
+                )
+
+        elif switch == "DC magnitude":
+            if is_dc_corrected:
+                # Overplot the levels of nearby source magnitudes
+                idx = pdf['i:fid'] == fid
+                if np.sum(idx):
+                    ref = np.mean(pdf['i:magnr'][idx])
```

Comment:
I wonder if `magnr` can change (beyond small fluctuations). The documentation states:
```
distance to nearest source in reference image PSF-catalog within 30 arcsec [pixels]
```
but is this catalog updated with new sources?

Reply:
I did not see any changes in their templates yet - that's why we still see these "always negative" objects from initial template transients. So for now magnr is just a constant. I thought about plotting it "properly" as a trace (using actual alert points) instead of purely horizontal line but that would make it less apparent (it will not span the whole plot)

Reply:
OK!